rskj-core/src/main/java/org/ethereum/vm/VM.java

```diff
@@ -699,16 +699,8 @@ protected void doCALLDATASIZE() {
     }
 
     protected void doCALLDATACOPY() {
-        DataWord size;
-        long newMemSize ;
-        long copySize;
-
         if (computeGas) {
-            size = stack.get(stack.size() - 3);
-            copySize = Program.limitToMaxLong(size);
-            checkSizeArgument(copySize);
-            newMemSize = memNeeded(stack.peek(), copySize);
-            gasCost += calcMemGas(oldMemSize, newMemSize, copySize);
+            gasCost += computeDataCopyGas();
```

Comment:
Any test to check the new gas cost for this opcode and the new RETURNDATACOPY opcode?

Reply:
It's not a new cost, I just extracted a method so I could reuse it from `RETURNDATACOPY `. I'll see if I can add a test for `RETURNDATACOPY `'s gas cost.